### PR TITLE
chore: Fill in package READMEs and update docs

### PR DESCRIPTION
**Pull request by jalling97**
- Moves local dev instructions out from the root `README.md` and into each packages' respective `README.md`
- Adds zarf deploy instructions to package READMEs
- Removes redundant instructions in some places
- Misc updates to package READMEs

### CODE REVIEW

**Review by CollectiveUnicorn — CHANGES_REQUESTED** [review]

File: packages/text-embeddings/README.md [hunk 1]
```diff
@@ -7,6 +7,33 @@ A LeapfrogAI API-compatible [instructor-xl](https://huggingface.co/hkunlp/instru
 
 # Usage
 
-:construction_worker: This documentation is still under construction. :construction_worker:
+## Zarf Package Deployment
 
+To build and deploy just the text-embeddings Zarf package (from the root of the repository):
 
+```shell
+uds deploy k3d-core-slim-dev:0.22.2      # if no cluster exists already
```
[the remark this answers is not in the document]
Comment (jalling97) [reply packages/text-embeddings/README.md:15]: Yeah the fewer the places that version number gets referenced the easier it'll be when we have to keep changing it.

**Review by justinthelaw — APPROVED** [review]
Great clean-up!

File: packages/text-embeddings/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jalling97) [reply packages/text-embeddings/README.md]: I ended up linking all the references to the cluster deployment back to the root README

**Review by CollectiveUnicorn — APPROVED** [review]
Looks good!

**Pull request merged**